google-cloud-storage/clirr-ignored-differences.xml

```diff
@@ -21,6 +21,16 @@
         <method>com.google.cloud.storage.BucketInfo$Builder deleteLifecycleRules()</method>
         <differenceType>7013</differenceType>
     </difference>
+    <difference>
```

Comment:
Yet again, we're getting stuck because of excessive use of interfaces. Either there are multiple implementations of Storage and we shouldn't make this change or we should bite the bullet and combine Storage and StorageImpl once and for all. Ditto StorageRpc and HttpStorageRpc.

Reply:
@frankyn what do you think about combining Storage and StorageImpl?